webui/react/src/components/kit/NoteCard.tsx

```diff
@@ -139,11 +137,11 @@ const NotesCard: React.FC<Props> = ({
         )
       }
       headStyle={{ marginTop: '16px', minHeight: 'fit-content', paddingInline: '16px' }}
-      style={{ ...style }}
       title={
         <Input
           defaultValue={title}
           disabled={!onSaveTitle || disabled}
+          style={{ width: '99%' }}
```

Comment:
Added a little space between title input and the edit icon

Reply:
can we add margin to the input instead or something like that? I'm not 100% sure using percentage based widths will scale.